src/dir.targets

```diff
@@ -2,6 +2,10 @@
   <PropertyGroup>
     <ErrorIfBuildToolsRestoredFromIndividualProject Condition="!Exists('$(ToolsDir)')">true</ErrorIfBuildToolsRestoredFromIndividualProject>
   </PropertyGroup>
+
+  <PropertyGroup Condition="$(NuGetTargetMoniker.StartsWith('.NETStandard,'))">
```

Comment:
Add a comment about why this is needed:
> The NETStandard.Library targets expect the TargetFrameworkIdentifier property to be set.  This is normally set the SDK targets by the but it isn't set by BuildTools.

I filed https://github.com/dotnet/buildtools/issues/1465 to try to get clarity on this moving forward.